src/transformers/models/nllb/tokenization_nllb_fast.py

```diff
@@ -148,10 +148,11 @@ def __init__(
         mask_token="<mask>",
         src_lang=None,
         tgt_lang=None,
-        additional_special_tokens=None,
+        additional_special_tokens=FAIRSEQ_LANGUAGE_CODES,
```

Comment:
We shouldn't have a mutable as a default argument here. If someone modifies this dictionary in some way, it'll change for all instances of the class. Instead we should do something like: 

```suggestion
        additional_special_tokens=None,
        ...
    ):
        if additional_special_tokens is None:
            additional_special_tokens = FAIRSEQ_LANGUAGE_CODES
```

Reply:
Great catch thanks 🤗